pennylane/beta/devices/default_tensor_tf.py

```diff
@@ -174,6 +168,15 @@ def __init__(self, wires, shots=1000, representation="exact", contraction_method
             contraction_method=contraction_method,
         )
 
+    @classmethod
+    def capabilities(cls):
+        capabilities = super().capabilities().copy()
+        capabilities.update(
+            provides_jacobian=True,
```

Comment:
"tensor_observables" not needed because it is inherited from ``QubitDevice``.

Reply:
Wait, doesn't `DefaultTensor` inherit from `Device`? (hence `tensor_observables` would not be defined)

Reply:
Correct, sorry :)